src/components/HeadingForm.js

```diff
@@ -46,8 +46,9 @@ class HeadingForm extends Component {
   formIsValid() {
     const { headingTitle } = this.state;
     const titleValid = headingTitle && headingTitle.length > 0;
+    const childOfValid = this.state.headingChildOf.length > 0;
 
-    return titleValid;
+    return titleValid && childOfValid;
```

Comment:
When adding a new heading; after validating the title the `Save` is enabled when `Child Of` option is empty. And if `Save` is clicked without choosing the `Child Of` options it saves the heading to the top of the structure (as 1st child of the root element). This check keeps the `Save` button disabled until the user selects the `Child Of` option.

@adamjarling Was this the expected behavior (before this additional check)?

Reply:
@Dananji @cjcolvar @bkeese  It depends how the data structure comes down.  Chris or Brian, should a user be able to add another top level element to the data structure, or will all additions be underneath the parent?

Reply:
![top-level-highlight](https://user-images.githubusercontent.com/3020266/55498987-84409f00-560a-11e9-91d9-658cf6744595.png)

Reply:
@adamjarling When `Child Of` option is not selected the new heading is added between `utah_phillips_one.222` and `Section 1` without this check. I think we don't have an option to add a heading on top of the existing top element (`utah_phillips_one.222`).
Sorry my previous comment was not clear.